src/metrics-modeling/tests/test_grading.py

```
@@ -0,0 +1,232 @@
+    report = grade_week(predictions, scores)
+
+    assert report.week_contests == 2
+    assert report.graded == 0
+    assert report.ungradeable == 2
+    assert report.su == {} and report.ats == {} and report.margin == {}
+
+    text = format_report(report, "BACKTEST empty")
+    assert "0 graded" in text
+
+
+def _fake_config():
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_fake_config`

(ANN202)